instabug/datadog_checks/instabug/__about__.py

```diff
@@ -0,0 +1,2 @@
+
+__version__ = '0.0.1'
```

Comment:
```suggestion
__version__ = '1.0.0'
```

This will be the first released version 🙂